### PR TITLE
Improve StorIOContentResolver Javadocs

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Huge pack of improvements to `StorIOContentResolver` Javadocs.

1. Dots at the end of each sentence.
2. Links and formatting.

@nikitin-da PTAL

### CODE REVIEW

**Comment by nikitin-da** [comment]
Delete emission. May be I made one more bug with concurrency =(

**Comment by artem-zinnatullin** [comment]
I don't think that it's your fail, probably, I made bug in notifications implementation…

Main problem — I can not reproduce it on my PC and laptop :(

**Comment by artem-zinnatullin** [comment]
I think I've found a problem — current implementation of notifications in `StorIOSQLite` is not fully thread safe, it can emit more notifications than receive.

**Comment by artem-zinnatullin** [comment]
Fixed `@see {@link SomeClass}` and replaced `<code></code>` to `{@code }`.

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutContentValuesIterable.java [hunk 1]
```diff
@@ -46,9 +49,18 @@
     }
 
     /**
-     * Creates {@link Observable} which will perform Put Operation and send results to observer
+     * Creates {@link Observable} which will perform Put Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutContentValuesIterable.java]: Quotes for `Cold` are missing =)

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutObject.java [hunk 2]
```diff
@@ -24,12 +30,31 @@ protected PreparedPutObject(@NonNull StorIOContentResolver storIOContentResolver
...
     @NonNull
     @Override
     public PutResult executeAsBlocking() {
         return putResolver.performPut(storIOContentResolver, object);
     }
 
+    /**
+     * Creates {@link Observable} which will perform Put Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutObject.java]: The same question about quotes. Or it isn't required?

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutObjects.java [hunk 3]
```diff
@@ -50,9 +50,18 @@
     }
 
     /**
-     * Creates {@link Observable} which will perform Put Operation and send result to observer
+     * Creates {@link Observable} which will perform Put Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutObjects.java]: Quotes?!

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/StorIOContentResolver.java [hunk 4]
```diff
@@ -20,45 +20,63 @@
...
 
     /**
-     * Subscribes to changes of required Uris
+     * Subscribes to changes of required Uris.
      *
-     * @param uris set of {@link Uri} that should be monitored
-     * @return {@link Observable} of {@link Changes} subscribed to changes of required Uris
+     * @param uris set of {@link Uri} that should be monitored.
+     * @return {@link Observable} of {@link Changes} subscribed to changes of required Uris.
+     * <p>Notice, that returned {@link Observable} is Hot Observable, which means,
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/StorIOContentResolver.java]: Quotes?!

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/StorIOContentResolver.java [hunk 5]
```diff
@@ -20,45 +20,63 @@
...
 
     /**
-     * Subscribes to changes of required Uri
+     * Subscribes to changes of required Uri.
      *
-     * @param uri {@link Uri} that should be monitored
-     * @return {@link Observable} of {@link Changes} subscribed to changes of required Uri
+     * @param uri {@link Uri} that should be monitored.
+     * @return {@link Observable} of {@link Changes} subscribed to changes of required Uri.
+     * <p>Notice, that returned {@link Observable} is Hot Observable, which means,
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/StorIOContentResolver.java]: Quotes?!

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteByQuery.java [hunk 6]
```diff
@@ -38,9 +38,18 @@ public DeleteResult executeAsBlocking() {
     }
 
     /**
-     * Creates {@link Observable} which will perform Delete Operation and send result to observer
+     * Creates {@link Observable} which will perform Delete Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteByQuery.java]: Quotes?!

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObject.java [hunk 7]
```diff
@@ -35,9 +39,18 @@ public DeleteResult executeAsBlocking() {
     }
 
     /**
-     * Creates {@link Observable} which will perform Delete Operation and send result to observer
+     * Creates {@link Observable} which will perform Delete Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObject.java]: Quotes?!

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObjects.java [hunk 8]
```diff
@@ -49,9 +49,18 @@
     }
 
     /**
-     * Creates {@link Observable} which will perform Delete Operation and send results to observer
+     * Creates {@link Observable} which will perform Delete Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObjects.java]: Quotes?!

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutContentValues.java [hunk 9]
```diff
@@ -36,9 +36,18 @@ public PutResult executeAsBlocking() {
     }
 
     /**
-     * Creates {@link Observable} which will perform Put Operation and send result to observer
+     * Creates {@link Observable} which will perform Put Operation and send result to observer.
+     * <p>
+     * Returned {@link Observable} will be Cold Observable, which means that it performs
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/put/PreparedPutContentValues.java]: Quotes?!

**Comment by artem-zinnatullin** [comment]
Fixed and rebased onto master.

**Pull request merged**